src/App.js

```diff
@@ -788,18 +789,39 @@ export default class App extends Component {
     const response = await fetch(endpoint);
     const body = await response.json();
     const {
-      proposals: proposalsList,
+      proposals,
       voteEndTime,
-      voteStartTime
+      voteStartTime,
+      trashAddress
     } = body.contents;
+
+    const proposalsList = proposals.map((proposal, i)=>{
+      return {
+        ...proposal,
+        _id: i
+      }
+    });
+
+    console.log({proposalsList})
+
+    const proposalsDictionary = proposalsList.reduce((acc,item)=>{
```

Comment:
Why it is needed? You can just pass proposal object into VoteControls (well, I already did this and it works fine)

Reply:
This code existed before you raised a PR. I was needed to pass proposal based on it's id.